pyteal/ast/subroutine.py

```diff
@@ -1014,12 +1014,12 @@ def __call__(self, subroutine: SubroutineDefinition) -> SubroutineDeclaration:
         proto = self.__proto(subroutine)
 
         args = subroutine.arguments()
-        arg_vars: list[ScratchVar] = []
+        arg_vars: list[tuple[ScratchVar, int]] = []
```

Comment:
```suggestion
        argvar_n_frameidx: list[tuple[ScratchVar, int]] = []
```
nit... suggest a renaming like this that reminds us that there's now a second coordinate

Reply:
👍